charts/sn-platform-slim/templates/detector/pulsar-detector-deployment.yaml

```diff
@@ -111,6 +111,10 @@ spec:
           -{{ $key }}
           {{- end }}
           {{- end }}
+          {{- if .Values.auth.authentication.enabled }} 
```

Comment:
This should only work for condition `.Values.auth.authentication.provider=jwt`?

Reply:
@ciiiii PTAL again.